Whiskey/Functions/Import-WhiskeyPowerShellModule.ps1

```diff
@@ -0,0 +1,54 @@
+
+function Import-WhiskeyPowerShellModule
+{
+    <#
+    .SYNOPSIS
+    Imports a PowerShell module.
+
+    .DESCRIPTION
+    The `Import-WhiskeyPowerShellModule` function imports a module into the global scope. The module must be installed in Whiskey's PowerShell modules directory. Use the `RequiresTool` attribute on a task to have Whiskey install a module in this directory.

```

Comment:
I don't wrap them because PowerShell's help command wraps them for you. If you wrap them manually, you have to make sure they wrap at a reasonable line length so they display nice in the console. I prefer to let the help command do that. When I'm reading/writing them, I usually use an editor that auto wraps.

Reply:
One counter point to that is that a lot of the Whiskey functions are internal only, where you wouldn't be using `Get-Help` to read their docs.